app/actions/websocket/users.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {fetchMe} from '@actions/remote/user';
+import DatabaseManager from '@database/manager';
+import {queryCurrentUser} from '@queries/servers/user';
+
+export async function handleUserUpdatedEvent(serverUrl: string, msg: any) {
+    const database = DatabaseManager.serverDatabases[serverUrl];
+    if (!database) {
+        return;
+    }
+    const currentUser = await queryCurrentUser(database.database);
+    const user = msg.data.user;
+
+    if (user.id === currentUser?.id) {
+        if (user.update_at > (currentUser?.updateAt || 0)) {
+            // Need to request me to make sure we don't override with sanitized fields from the
+            // websocket event
+            fetchMe(serverUrl, false);
```

Comment:
I really despise this, but it seems there is no workaround, right ?

Reply:
Just for my understanding, why do we `fetchMe` if the WS event would have the latest information?

Reply:
The server sends a sanitized profile in the websocket. That means that needed information like the "NotifyProps" is not present, and:
- We may overwrite it if not careful
- May be the changed value, and we need to handle the changes
To deal with all that as a whole we just fetchMe from the server, and make sure we have all the values (included the sanitized ones) up to date.

Reply:
ok let's have an eye on https://mattermost.atlassian.net/browse/MM-40582 and we can determine the server version before fetching once that ticket is implemented.

cc: @agnivade